wml_test_schedule

```diff
@@ -160,6 +160,7 @@
 0 event_test_filter_attack
 0 event_test_filter_attack_no_defense
 0 event_test_filter_attack_type
+0 event_test_filter_attack_type_fail
```

Comment:
The suffix `_fail` should be reserved for tests which return a non-zero status. Please rename it.

Reply:
Because bug in has_attack filter, i sould probably close this Pr and same remove replacement_type  for 1.20

Reply:
name changed